src/makielayout/interactions.jl

```diff
@@ -157,8 +157,10 @@ function process_interaction(r::RectangleZoom, event::MouseEvent, ax::Axis)
     end
 
     if event.type === MouseEventTypes.leftdragstart
-        data = Makie.apply_transform(inv_transf, event.data)
-        prev_data = Makie.apply_transform(inv_transf, event.prev_data)
+        # because these are designed to alter the axis, they 
+        # are kept in data space. 
+        data = Makie.apply_transform(inv_transf, event.data, :data)
+        prev_data = Makie.apply_transform(inv_transf, event.prev_data, :data)
```

Comment:
I think all of the ones in this file can just continue being `Makie.apply_transform(inv_transf, event.data)` etc.

Reply:
I reverted all the changes to this file.